src/api_manager/check_auth.cc

```diff
@@ -170,6 +171,10 @@ void AuthChecker::Check() {
 void AuthChecker::GetAuthToken() {
   Request *r = context_->request();
   std::string auth_header;
+  if (r->FindHeader(KAuthHeaderIAP, &auth_header)) {
```

Comment:
second issue is order or precedence or both authorization and this one is set.  
I think we should take "authorization" first.  if not there,  then check this one